ckanext/data_qld/resource_visibility/helpers.py

```diff
@@ -68,17 +68,17 @@ def process_resources(data_dict, user_obj):
         resources = data_dict.get('resources', [])
         # if user is NOT Member, Editor, Admin remove the resource.
         if not has_user_permission_for_org(data_dict.get('owner_org'), user_obj, 'read'):
-            # Loop each resources and remove resource
+            # Loop through each resource and remove the resource when the below condition is True
             # If resource_visible is `FALSE` or
             # resource_visible is `TRUE` and governance_acknowledgement is `NO` and de_identified_data is `YES`
             de_identified_data = data_dict.get('de_identified_data', 'NO')
             for resource in list(resources):
-                resource_visible = resource.get('resource_visible', 'FALSE')
-                governance_acknowledgement = resource.get('governance_acknowledgement', 'NO')
-                # Remove resource if the condition is True
-                hide_resource = resource_visible == 'TRUE' and governance_acknowledgement == 'NO' and de_identified_data == 'YES'
-                # Always remove if the `resource_visible` is `FALSE`
-                if (resource_visible == 'FALSE' or hide_resource):
+                hide_resource = resource.get('resource_visible', 'FALSE') == 'FALSE'
```

Comment:
The default value is "not visible". That seems questionable.

Reply:
Yeh that is not correct, I updated the default value to 'TRUE'